packages/babel-traverse/src/scope/lib/renamer.js

```diff
@@ -2,9 +2,33 @@ import Binding from "../binding";
 import splitExportDeclaration from "@babel/helper-split-export-declaration";
 import * as t from "@babel/types";
 
+function isInFunctionParameterScope(path) {
+  const computableNode = path.findParent(path => {
+    return (
+      path.type === "AssignmentPattern" ||
+      (path.type === "ObjectProperty" && path.node.computed)
+    );
+  });
+  const fdPath = path.findParent(path => path.type === "FunctionDeclaration");
```

Comment:
1. We should only compute this is `computableNode` has been found
2. Can we share lines 12-17 with the same code in the `isScopeInFunctionParameter` function?